### ISSUE TITLE
Dev Branch. Not an Issue

### ISSUE THREAD

**Issue by kevdagoat** [nutterpc/tg-luci#10]
Can be found here:
https://github.com/nutterpc/tg-luci/tree/dev

**Comment by 0jay** [comment]
The list referred to in the guide is a list of modules that wont segfault
the device.
No such list is provided.



2018-09-30 02:16:18 +0000 Kevdagoat <notifications@github.com> :

**Comment by kevdagoat** [comment]
I didn’t change that at all

**Comment by 0jay** [comment]
yeah what I’m saying is if a list is mentioned then one should be provided.
alternatively you should indicate that one will be provided in the future.

these options are opposed to mentioning a list that either doesn’t exist or
is private.



2018-09-30 02:58:35 +0000 Kevdagoat <notifications@github.com> :
